.pre-commit-config.yaml

```diff
@@ -17,3 +17,17 @@ repos:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+-   repo: local
```

Comment:
Please explain how this works. I didn't look into it too deeply, but my first impression was that `local` meant local to your machine. Will this still work in the library when someone PRs to it?

Reply:
Right, it means that it's not pulling the code that pre-commit runs from another repo, but rather, it's executing things locally.

Since part of the GitHub workflow is to install pylint, and that happens before pre-commit runs when someone creates a PR against a GitHub repository, we will be assured that pylint is installed.

The downside is that pylint isn't automatically installed as it would with pre-commit. If we want, we _could_ add that to the custom hook, however, I know we'd rather not pollute peoples' environments by installing things willy-nilly.